man/systemd.network.xml

```diff
@@ -1910,6 +1910,16 @@
         <listitem><para>Similar to the <varname>DNS=</varname> settings described above, these
         settings configure whether and what POP3 server information shall be emitted as part of
         the DHCP lease. The same syntax, propagation semantics and defaults apply as for
+        <term><varname>SMTPServers=</varname></term>
+        <varname>DNS=</varname>.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>SMTPServers=</varname></term>
+
+        <listitem><para>Similar to the <varname>DNS=</varname> settings described above, these
```

Comment:
DNS= settings → DNS= setting
these settings configure → this setting configures
"." missing before "The same syntax"